sevntu-checks/sevntu-checks.xml

```diff
@@ -80,6 +80,10 @@
       <property name="forbiddenClassesRegexpProtected" value=""/>
       <property name="forbiddenClassesRegexpPublic" value=""/>
     </module>
+    <module name="com.github.sevntu.checkstyle.checks.coding.ForbidCertainMethodCheck">
+      <property name="methodName" value="exit"/>
+      <property name="argumentCount" value=""/>
```

Comment:
Empty value should result in configuration exception.
Please set 0 here and please add UT on such case.

Reply:
I have put 1 as System.exit takes 1 argument.